daemon/logger/alilogs/logapi.go

```diff
@@ -0,0 +1,72 @@
+// Pakcage alilogs api interface
+
+package alilogs
+
+import (
+	"errors"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/galaxydi/go-loghub"
+)
+
+// AliLogAPI define log api interface
+type AliLogAPI interface {
+	PutLogs(*sls.LogGroup) error
+}
+
+// AliLogClient implements AliLogAPI interface
+type AliLogClient struct {
+	Endpoint        string
+	ProjectName     string
+	LogstoreName    string
+	accessKeyID     string
+	accessKeySecret string
+	sessionToken    string
+	project         *sls.LogProject
+	logstore        *sls.LogStore
+}
+
+// PutLogs implements ali PutLogs method
+func (client *AliLogClient) PutLogs(logGroup *sls.LogGroup) error {
+	return client.logstore.PutLogs(logGroup)
+}
+
+// NewAliLogClient ...
+func NewAliLogClient(serviceEndpoint, projectName, logstoreName, accessID, accessSecret, token string) (AliLogAPI, error) {
+	client := AliLogClient{}
+	client.Endpoint = serviceEndpoint
+	client.ProjectName = projectName
+	client.LogstoreName = logstoreName
+	client.accessKeyID = accessID
+	client.accessKeySecret = accessSecret
+	client.sessionToken = token
+
+	logrus.WithFields(logrus.Fields{
+		"endpoint":     serviceEndpoint,
+		"projectName":  projectName,
+		"logstoreName": logstoreName,
+	}).Info("Created alilogs client")
```

Comment:
Move this line to the end of this method since the client hasn't been fully populated.